server/server.go

```diff
@@ -105,6 +108,28 @@ func handler(s ssh.Session) {
 	verbose("handler exits")
 }
 
+func Advertise(hostname, port, txt string) (*mdns.Server, error) {
+	v("starting mdns server")
+	// Setup our service export
+	host, _ := os.Hostname()
```

Comment:
Is it safe to ignore the error here? I think you might want to handle it?

Reply:
(Specifically referring to the error returned by `os.Hostname()`)

Reply:
It's a fair comment.  In some ways I was trying to follow lighter weight error handling focus elsewhere in the code -- but this sort of error likely to cause issues so maybe better explicitly report and then bail out.